app/models/assignment.js

```diff
@@ -34,6 +34,47 @@ export default class AssignmentModel extends Model {
   // see the my projects controllers
   sortingActualEndDate;
 
+  // if the each dcpPublichearinglocation and dcpDateofpublichearing properties are filled in dispositions array,
+  // then hearings have been submitted for that project
+  @computed('dispositions.@each.{dcpPublichearinglocation,dcpDateofpublichearing}')
+  get hearingsSubmitted() {
+    const dispositions = this.get('dispositions');
+    // array of hearing locations
+    const dispositionHearingLocations = dispositions.map(disp => `${disp.dcpPublichearinglocation}`);
+    // array of hearing dates
+    const dispositionHearingDates = dispositions.map(disp => disp.dcpDateofpublichearing);
+    // hearingsSubmittedForProject checks whether each item in array is truthy
+    const hearingsSubmittedForProject = dispositionHearingLocations.every(location => !!location) && dispositionHearingDates.every(date => !!date);
+    return hearingsSubmittedForProject;
+  }
```

Comment:
This looks familiar... seems like we need this kind of validation check on multiple things? I like that it's here on the model.

Reply:
Are you referring to seeing it also on the new hearings-list-on-milestones-list component?  `hearingSubmitted` here checks for `assignment.dispositions` whereas `hearingsSubmitted` in the hearings-list-on-milestones-list checks `project.dispositions` associated with a particular milestone -- so would be validating two different things. Maybe I can move `hearingsSubmitted` that's on the hearings-list-on-milestone-list component to the `milestone` model ?